scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/rmproxies/RMProxyActiveObject.java

```diff
@@ -143,15 +151,17 @@ public void releaseNodes(NodeSet nodes, Script<?> cleaningScript, Map<String, Se
     /**
      * Execute the given script on the given node.
      * Also register a callback on {@link #cleanCallBack(Future)} method when script has returned.
-     *
      * @param node           the node on which to start the script
      * @param cleaningScript the script to be executed
      * @param variables
+     * @param genericInformationOverridden
```

Comment:
"genericInformationOverridden" is a bit overkill, simply name this parameter genericInformation